src/AppInstallerCLICore/Commands/SearchCommand.cpp

```diff
@@ -79,7 +80,18 @@ namespace AppInstaller::CLI
             Workflow::OpenSource() <<
             Workflow::SearchSourceForMany <<
             Workflow::HandleSearchResultFailures <<
-            Workflow::EnsureMatchesFromSearchResult(false) <<
-            Workflow::ReportSearchResult;
+            Workflow::EnsureOneMatchFromSearchResult(false);

```

Comment:
Moved 1 more workflow up than needed
For list versions, we should use Workflow::EnsureOneMatchFromSearchResult(false)
else, Workflow::EnsureMatchesFromSearchResult(false)